src/main/java/ch/njol/skript/expressions/ExprFormatTime.java

```diff
@@ -40,7 +40,7 @@
 import ch.njol.util.Kleenean;
 
 @Name("Formatted time")
```

Comment:
You should also fix titles like this
```suggestion
@Name("Formatted Time")
```